go/gosec/traversal/zip.go

```diff
@@ -0,0 +1,46 @@
+package unzip
+
+import (
+	"archive/zip"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func unzip(archive, target string) error {
+	// ruleid: path-traversal-inside-zip-extraction
```

Comment:
Can we add a True Negative test as well?

Reply:
Will do!